src/preact/base-element.js

```diff
@@ -139,6 +143,14 @@ export class PreactBaseElement extends AMP.BaseElement {
 
     this.defaultProps_ = this.init() || null;
 
+    const useContexts = Ctor['useContexts'];
+    if (useContexts.length != 0) {
+      subscribe(this.element, useContexts, (...contexts) => {
+        this.contextValues_ = contexts.slice(0);
```

Comment:
`contexts` is already a brand new array here, so no need to slice a copy.
```suggestion
        this.contextValues_ = contexts;
```

Reply:
Makes sense.